docs/sites.yml

```diff
@@ -3817,11 +3817,24 @@
     - API
     - Documentation
   featured: false
+- title: Sabbiu Shah
+  main_url: https://sabbiushah.com.np
+  url: https://sabbiushah.com.np
+  source_url: https://github.com/sabbiu/sabbiushah.com.np
+  description: >
+    Sabbiu Shah is a personal blog written by me. Blogs are written on Artificial Intelligence, Mathematics, Economics. Visual explanation and codes are also in the blog.
+  categories:
+    - Blog
+    - Education
+    - Open Source
+  featured: false
+  built_by: Sabbiu Shah
+  built_by_url: https://twitter.com/sabbiushah
 - title: klarna
   url: https://developers.klarna.com/en/gb/kco-v3
   main_url: https://developers.klarna.com/en/gb/kco-v3
   description: Here you will find the integration guides, API documentation, and more to easily integrate with Klarna.
   categories:
     - API
     - Documentation
-  featured: false
+  featured: false
```

Comment:
Take a look at [.editorconfig](https://github.com/gatsbyjs/gatsby/blob/20e6bce47fcb4cadda8449603d1472a2bad37b3e/.editorconfig#L6), I think maybe we need a new line here for consistency.